adoc/Harbor-Private-Registry-Administration.adoc

```diff
@@ -0,0 +1,348 @@
+:experimental:
+
+[#administration]
+== Administration
+
+=== Setup Administration Access
+
+{spr} User Interface can be accessed from a supported web browser at the location provided as `<ingress_url>` during the installation.
+Find out about the initial admin user password in <<install-passwords>.
+
+image::registry-harbor-landingpage.png[scaledwidth=80%]
+
+After the first login, you can change the administrator's password through the web UI. Select the *admin* tab and select menu:admin[Change Password].
```

Comment:
Too many _admin_s here. As you can see in the picture, there's only one menu item call _admin_ and then there's Change Password.